### PR TITLE
Rename history to events

### PR DESCRIPTION
**Pull request by jasollien**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1926,66 +1926,71 @@ Retrieves a document as binary file.
...
+|description_updated|The description (limit: 1024 characters)|
+|description_removed|null|
+|status_updated|The status (value from extensions) |
+|type_updated|The type (value from extensions)|
+|priority_updated|The priority (value from extensions)|
+|priority_removed|null|
+|due_date_updated|The due date|
+|due_date_removed|null|
+|assigned_updated|The assigned user (value from extensions)|
+|assigned|updated, removed|null|
```
Comment (GeorgDangl) [inline README.md]: Should it be `|assigned_removed|null|`?

**Comment by GeorgDangl** [comment]
Didn't find anything else besides maybe changing the names of the `topic.assigned_to` operators from `assigned_updated`/`removed` to `assigned_to_updated`/`removed` (for consistency).

File: README.md (hunk 1 above)
Comment (jasollien) [inline README.md]: yes :)

**Comment by jasollien** [comment]
I agree on that. I will fix

**Pull request merged**